Sources/Server/Response.swift

```diff
@@ -72,6 +72,55 @@ public class Response {
         let header = "HTTP/1.1 302\r\nContent-Length: \(body.utf8.count)\r\nContent-type: text/html\r\nLocation: /\r\n\r\n" 
         return header + body
     }
+
+    private func handleForm(request: Request) -> String {
+        let filePath = NSURL.fileURL(withPath: "log.txt")
```

Comment:
This file is not a log file; it's a data file. Your log file (you should have one) should only be for writing events to.